app/containers/App.js

```diff
@@ -1,23 +1,23 @@
-import React, { Component, PropTypes } from 'react'
+import React from 'react'
+const DevTools = require('./DevTools')
 
-export default class App extends Component {
-  static propTypes = {
-    children: PropTypes.element.isRequired,
-  }
+const App = (props) => {
+  return (
+    <div>
+      { props.children }
+      {
+        (() => {
```

Comment:
This one I'd like to test against a current version before I submit changes because I'm currently not getting Redux DevTools to work at all. Sadly I am running into #81 now on a fresh install (it's the electron asar module version mismatch thing all over again - I'm on Linux) so I'll revert back to master.

FWIW, [here's the recommended setup](https://github.com/gaearon/redux-devtools/blob/master/docs/Walkthrough.md#containersrootjs) for DevTools, would be nice and clean IMO if it works